Formula/antidote.rb

```diff
@@ -0,0 +1,28 @@
+class Antidote < Formula
+  desc "Get the cure to slow zsh plugin management"
+  homepage "https://getantidote.github.io/"
+  url "https://github.com/mattmc3/antidote/archive/refs/tags/v1.4.1.tar.gz"
+  sha256 "b87804b55fb6a33aac4864d983b216f6fae3b0ccf26fa3d5ce3e8d123133fff8"
+  license "MIT"
+
+  uses_from_macos "zsh"
+
+  def install
+    pkgshare.install "antidote.zsh"
+    pkgshare.install "functions"
+    rm_f Dir["man/*.md"]
+    man.install Dir["man/*"]
+  end
+
+  def caveats
+    <<~EOS
+      To activate antidote, add the following to your ~/.zshrc:
+        source #{HOMEBREW_PREFIX}/share/antidote/antidote.zsh
```

Comment:
```suggestion
        source #{pkgshare}/antidote.zsh
```

Reply:
Good tip. I used antigen and antibody as my template, which did it the old way:
- https://github.com/Homebrew/homebrew-core/blob/master/Formula/antigen.rb
- https://github.com/Homebrew/homebrew-core/blob/master/Formula/antibody.rb

I've pushed a new commit.